### PR TITLE
add iftip.eth

### PR DESCRIPTION
**Pull request by bibodeng**
#### Hi! What is your PR about?

- [x] Add images for your space
- [x] Add custom domain for your space
- [ ] Add alias for migrate your space
- [ ] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/iftip.eth/domains.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "vote.iftip.io": "iftip.eth"
+}
```
Comment (gokaiorg) [inline spaces/iftip.eth/domains.json:3]: - You have to add your subdomain in the list of the file https://github.com/snapshot-labs/snapshot-spaces/blob/master/spaces/domains.json
- You should not add domains.json files in your space, please delete it.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/iftip.eth/domains.json (hunk 1 above)
Comment (bibodeng) [reply spaces/iftip.eth/domains.json]: OK, I've done this in new fix update

**Comment by gokaiorg** [comment]
To update your fork with the remote version of snapshot-spaces you can try these GIT commands then commit your changes again.
`git remote add upstream https://github.com/snapshot-labs/snapshot-spaces.git`
`git fetch upstream`
`git checkout master`
`git merge upstream/master`

**Comment by bibodeng** [comment]
@gokaiorg Hi, I fix the conflict, you can go ahead

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**